specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -3182,6 +3191,40 @@
         }
       },
       "description": "Properties of a private link resource."
-    }
+    },
+    "EncryptionProperty": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Indicates whether or not the encryption is enabled for the workspace.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "EncryptionStatus",
+            "modelAsString": true
+          }
+        },
+        "keyVaultProperties": {
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Customer Key vault properties."
+        }
+      }
+    },
+    "KeyVaultProperties": {
+     "type": "object",
+     "properties": {
+       "keyIdentifier": {
+         "description": "Key vault uri to access the encryption key.",
+         "type": "string"
+       },
+       "identityClientId": {
```

Comment:
You can remove this for now since you don't support userAssigned identities.  It can be added to your implementation of the API when that comes in if necessary.